src/orga/getting-in-touch.rst

```diff
@@ -3,14 +3,19 @@
 Getting in Touch
 ================
 
-There are many ways to get in touch with the conda-forge community, primarily
-through issue trackers, mailing lists, and real-time chat.
+The community at conda-forge would be happy to connect with you. We have got several ways for you to communicate with us, primarily through issue trackers, mailing lists, and real-time chat.
```

Comment:
this first phrase "The community at conda-forge would be happy to connect with you." and the rest of the text seem a little strange in my opinion... maybe this at the end of the "we have got..."?

Reply:
Oh! I was thinking to make it sound more welcoming.

Reply:
Indeed it is, and I liked that! is just that by some reason those "with you" and "with us" together sound strange...

Reply:
Should I go like : The community at conda-forge would be happy to connect with you. We have got several ways for you to communicate, primarily through issue trackers, mailing lists, and real-time chat.

Or do i need to rephrase this?